amnesia/src/CMakeLists.txt

```diff
@@ -48,6 +48,19 @@ add_compile_options(
     "$<$<COMPILE_LANGUAGE:CXX>:-Wno-delete-incomplete>"  # Suppress warnings about deleting incomplete types
 )
 
+# Define LINUX if we are on a Linux system
+if(UNIX AND NOT APPLE)
+    set(LINUX TRUE)
```

Comment:
This variable is normally set in BoilerPlate and I'm not sure duplicating it here is a good idea - I assume it was added for the warning below and not because it does not propagate to the launcher or the game?

You can just add the warning suppression for all builds along with the previous ones. It's a single occurrence of that warning too, so should be easy to fix in the source code later on.

Reply:
Sidenote, I do feel like BoilerPlate stuff should be properly available in a global context. At some point I will review it and make it global and set in the src build instead of the hpl2 build, but it's another pet peeve edit that could wait until more important things are resolved - such as the MSVC build or building all libs from source on MinGW...